### DISCUSSION TITLE
I build double bus station why my bus keep load and unload only in one side?

### DISCUSSION THREAD

**Discussion by mricebear** [voxeltycoon/issues#895] [General] [answered]
![20210906185729_1](https://user-images.githubusercontent.com/87141731/132214375-0dd0edfe-a585-427c-9f26-75a2ff7c4aab.jpg)

PLS check this out,

I need some explanation how this double station bus working

**Comment by victorleblais** [comment, marked as answer]
It just depends on which direction they come from. Vehicles will try to take the shorter path to their destination so the side of the station they use will depend on which direction they come from.

Not to plug my own mods, but I made additional bus and truck stops:
https://steamcommunity.com/sharedfiles/filedetails/?id=2469760727

And with my custom stops, you can pick which side of the road you want the bus to use, which can be helpful in some cases.
